src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/blackduck/FullScanPostProcessingOperation.java

```diff
@@ -41,6 +44,7 @@
 import com.synopsys.integration.rest.HttpUrl;
 
 public class FullScanPostProcessingOperation {
+    private final Logger logger = LoggerFactory.getLogger(getClass());
     private final DetectToolFilter detectToolFilter;
```

Comment:
is logger (still) used?

Reply:
It isn't anymore.  I just pushed a change to remove it.